cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -584,7 +584,6 @@
     // NOTE: "v3" to avoid conflict with data of older metrics system
     public static final String DEFAULT_METRIC_V3_TABLE_PREFIX = "metrics.v3.table";
     public static final String METRICS_HBASE_MAX_SCAN_THREADS = "metrics.hbase.max.scan.threads";
-    public static final String METICS_HBASE_TABLE_SPLITS = "metrics.v3.table.splits";
```

Comment:
On second thought, this number should be configurable in the cConf. Larger clusters may need more than 16. I am not sure how this can be changed, though, once the tables are created. Sp perhaps we can get it from cConf when creating the table, store it in the table properties, and at runtime use the number from the properties. Sorry that is just how you had it initially. But we have to make sure the dataset properties cannot be changed, so MetricsTableDefinition.reconfigure() must throw an exception if it changes. 

This does not have to be in this PR, can you open a Jira for making metrics table splits configurable?